src/settings-compat-v7/_font.scss

```diff
@@ -0,0 +1,28 @@
+// --- Fonts and typography --- //
+
+// Font weights
+$fw-semi-bold: 600 !default;
+
+// Font sizes
```

Comment:
IMHO these var declarations should be as follows:
```
// Font sizes
 +$fz-base: 14 !default;
 +$fz-body: $fz-base * 1px;
 +$fz-xs: ceil($fz-base * .666) * 1px;
 +$fz-s: ceil($fz-base * .8) * 1px;
 +$fz-m: $fz-base * 1px;
 +$fz-l: ceil($fz-base * 1.25) * 1px;
 +$fz-xl: ceil($fz-base * 1.75) * 1px;
```
So, every site might redefine its own `$fz-base` but *will not* be able to change the ratio and proportion of font increase. These values should be predefined by UX and should not be overwritten.

Reply:
That's why src/settings-compat-v7/_font.scss is for. We'll make it completely effectless